pyproject.toml

```diff
@@ -261,6 +261,7 @@ ignore = [
 "tests/**.py" = [  # Some things we want for the moin project are unnecessary in tests.
     "D",  # Ignore docstring rules in tests
     "ANN", # Ignore type annotations in tests
+    "ARG001", # Ignore unused function arguments in tests (typically fixtures)
```

Comment:
Wouldn't it be useful to have this to warn about unused fixtures from regular test functions, and ignore locally when defining a fixture?

Reply:
removed this ignore